wire-ios-utilities/Source/Public/ZMAccentColor.h

```diff
@@ -16,16 +16,5 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-typedef NS_ENUM(int16_t, ZMAccentColor) {
-    ZMAccentColorUndefined = 0,
-    ZMAccentColorStrongBlue,
-    ZMAccentColorStrongLimeGreen,
-    ZMAccentColorBrightYellow,
-    ZMAccentColorVividRed,
-    ZMAccentColorBrightOrange,
-    ZMAccentColorSoftPink,
-    ZMAccentColorViolet,
-    
-    ZMAccentColorMin = ZMAccentColorStrongBlue,
-    ZMAccentColorMax = ZMAccentColorViolet,
-};
+extern int16_t const ZMAccentColorMin;
+extern int16_t const ZMAccentColorMax;
```

Comment:
Those values are only used for tests, so I would suggest to remove it and find another solution in those places.

<img width="1156" alt="Screenshot 2024-04-23 at 11 24 44" src="https://github.com/wireapp/wire-ios/assets/7196958/241370a6-2760-4da9-814c-90f5b1cfc75a">

Reply:
ZMUser.m accesses it as well. I'll postpone it for now.